### PR TITLE
Remove funds from IBC client SendFunds

### PR DESCRIPTION
**Pull request by CyberHoward**
This PR removes `funds` field from the IBC client SendFunds function. `SendFunds` now uses `msg_info.funds` instead.

### CODE REVIEW

**Comment by Kayanski** [comment]
Wait so the user can't sent funds that are located inside their account already ? Or in a complicated way only ?

**Comment by CyberHoward** [comment]
Sorry, description wasn't very clear. 

Instead of using the `funds` field specified within the message, this pr removes it to just use the `msg_info.funds` instead. That way we don't have to replicate that data.

File: framework/contracts/native/ibc-client/src/commands.rs [hunk 1]
```diff
@@ -397,7 +396,7 @@ pub fn execute_send_funds(
     let ics20_channel_id = ics20_channel_entry.resolve(&deps.querier, &ans)?;
 
     let mut transfers: Vec<CosmosMsg> = vec![];
-    for coin in funds {
+    for coin in info.funds.into_iter() {
```
Comment (Kayanski) [inline framework/contracts/native/ibc-client/src/commands.rs:399]:
```suggestion
    for coin in info.funds {
```

**Review by Kayanski — APPROVED** [review]
LGTM

**Pull request merged**